pkg/api/validate/dynamic/workersku.go

```diff
@@ -0,0 +1,53 @@
+package dynamic
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/util/computeskus"
+)
+
+func (dv *dynamic) ValidateWorkerSku(ctx context.Context, location string, subscriptionID string, oc *api.OpenShiftCluster) error {
```

Comment:
Would you please provide high level explanation what is the function validating and why?

Reply:
thanks for your review, I have added a little more detail to the comments.